pkgs/development/libraries/boehm-gc/default.nix

```diff
@@ -54,13 +59,12 @@ stdenv.mkDerivation rec {
       Alternatively, the garbage collector may be used as a leak detector for
       C or C++ programs, though that is not its primary goal.
     '';
-
-    homepage = "https://hboehm.info/gc/";
-
     # non-copyleft, X11-style license
+    changelog = "https://github.com/ivmai/bdwgc/blob/v${finalAttrs.version}/ChangeLog";
     license = "https://hboehm.info/gc/license.txt";
-
-    maintainers = [ ];
-    platforms = lib.platforms.all;
+    maintainers = with maintainers; [ AndersonTorres ];
+    platforms = platforms.all;
   };
-}
+
+  passthru.tests = nixVersions;
```

Comment:
not sure how I feel about passthru being after meta. Personally, i don't expect there to be anything after meta, has this changed?

Reply:
To be honest I don't know. I found both ways in a cursory look.
I believe that both are a bit "meta" in the sense they do not trigger rebuilds.
I will revert this.